src/pages/api/admin/loadClassInfo.ts

```diff
@@ -0,0 +1,29 @@
+import { ClassModel } from '@server/db/class'
+import { apiController } from '@utils/api'
+import { Types } from 'mongoose'
+
+export interface LoadClassInfoParams {
+  cid: string
+}
+
+export interface LoadClassInfoResults {
+  name: string
+  topics: string[]
+  students: Types.ObjectId[]
```

Comment:
We are using `ID` type to refer to id values.

Reply:
> We are using `ID` type to refer to id values.

we are using this just to count the number of students in the class not for anything else so I hope it is fine:)

Reply:
I meant the type definition is wrong. If all you need is student count, it should be `number` instead of `List`

Reply:
> I meant the type definition is wrong. If all you need is student count, it should be `number` instead of `List`

Oh that's right updated it here https://github.com/kixlab/kuiz/pull/46/commits/b51043c3294b662d0fb4a7331b61915ff6839fc7